### PR TITLE
Adding new ap-southeast-1 region

### PR DESCRIPTION
**Pull request by ddubeau**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: functions/source/rest_client.py [hunk 1]
```diff
@@ -352,7 +476,7 @@ def post_request(url, json_data, encodedbase64, user_agent, version):
     # extracting data in json format 
     data = resp.json() 
     
-    # If the response was successful, no Exception will be raised
+    # Raise and Exception if the response is unsuccessful
```
Comment (sshvans) [inline functions/source/rest_client.py:479]:
```suggestion
    # Raise an Exception if the response is unsuccessful
```

**Review by sshvans — COMMENTED** [review]
1 typo. Otherwise looks good overall. Are you able to test it using taskcat successfully in all the regions?
Also, I need to see how do we tackle 'hard coded account ids' error check.

**Comment by sshvans** [comment]
Also do you still need another PR #69 ?

**Comment by ddubeau** [comment]
> Also do you still need another PR #69 ?

Do you mean PR #68 because PR #69 also includes the APJ region?

**Pull request merged**